nzsl-dictionary-scripts/env-prerelease/main.tf

```
@@ -0,0 +1,72 @@
+
+terraform {
```

nitpick: unneeded whitespace:

```suggestion
terraform {
```